chain/checkpointing/constant.go

```diff
@@ -0,0 +1,7 @@
+package checkpointing
+
+var FEE float64 = 0.01
+var S3_HOST string = "minio.deadbrain.corp"
+var ACCESS_KEY_ID string = ""
+var SECRET_ACCESS_KEY string = ""
+var BUCKET_NAME string = "eudico"
```

Comment:
Suggestion for the future: what about adding these in the config? I don't expect these to be constants in every runtime. Each user will probably chose their own, right?

Reply:
Done!

I have added a special section with those value so they can de loaded at start. It still can be improved.